tests/storage/test_cdi_certificate.py

```diff
@@ -184,7 +184,6 @@ def test_dv_delete_from_vm(
 
 
 @pytest.mark.sno
-@pytest.mark.gating
```

Comment:
Why remove this from gating? Is it due to flakiness? If yes, please add a TODO.

Reply:
Yes, due to flakiness. I believe we will revisit the gating tests periodically, so wouldn't want to have TODO in the code

Reply:
Is there any jira tracker for this?

Reply:
No, no need for the jira ticket as well. This test is not qualified for gating.